tests/error-handling.test.ts

```
@@ -0,0 +1,316 @@
+
+    // Should not throw error even with malformed HTML
+    const result = await page.evaluate(() => {
+      const table = document.querySelector('table');
+      return table !== null;
+    });
+
+    expect(result).toBe(true);
+    await page.close();
+  });
+
+  test('should handle network timeouts appropriately', async ({}, testInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)